server/routes/reviews.js

```diff
@@ -42,4 +56,50 @@ router.route('/:product_id')
     }
   });
 
+router.route('/update')
+  .patch(async (req, res) => {
+    try {
+      let options = {review_id: req.body.review_id};
+      await updateReview(options, req.body);
+      res.sendStatus(200);
+    } catch {
+      res.status(500).send('Internal Server Error.');
+    }
+  });
+
+router.route('/add-review')
+  .post(async (req, res) => {
+    try {
+      checkRequestBody(req.body);
+      addReview(req.body);
+      res.sendStatus(200);
+    } catch {
+      console.log('there')
+      res.status(500).send('Internal Server Error.');
+    }
+  });
+
+router.route('/delete')
```

Comment:
I believe the final catch statement will catch any errors meaning that you don't need to have the other catch statements within your delete function. Also, I would see if you could clean up your code a little bit here. Since you're using async already you don't need to use .then, you should be using await syntax instead. Just a suggestion, as if this works fine for you then you can keep it as is.

`router.route('/delete')`
  `.delete(async (req, res) => {`
      `try {`
         ` const lastID = await getLastReviewId();`
        `  await deleteReview(req.body.review_id);`
         ` await updateReview({review_id: lastId[0].review_id}, {review_id: req.body.review_id});`
       `   res.sendStatus(200);`
   `   } catch(err) {`
        `    console.error(err)`
        `    res.status(500).send('Internal Server Error.');`
      `}`
`  }`